catkin_tools/common.py

```diff
@@ -399,6 +400,19 @@ def log(*args, **kwargs):
             unicode_error_printed = True
 
 
+__warn_terminal_width_once_has_printed = False
+
+
+def __warn_terminal_width_once():
+    global __warn_terminal_width_once_has_printed
+    if __warn_terminal_width_once_has_printed:
+        return
+    __warn_terminal_width_once_has_printed = True
+    print('WARNING: Could not determine the width of the terminal. '
+          'This warning will only be printed once.',
+          file=sys.stderr)
```

Comment:
In my latest commit it is passing, I think it was picking up the `WARNING` in a check for whether or not there were warnings 😄. So changing it to `NOTICE` was an easy workaround.